docs/config.yaml

```diff
@@ -8,7 +8,7 @@ googleAnalytics: UA-104956980-3
 pygmentsCodefences: true
 pygmentsStyle: "pygments"
 enableGitInfo: true
-
+resourceDir: docs/resources
```

Comment:
Is this change needed? I'm assuming it is just to let Hugo know where to look for static assets? I think we use static url to get file locations.